artifactory-pro/plan.sh

```diff
@@ -1,20 +1,18 @@
 pkg_origin=core
 pkg_name=artifactory-pro
-pkg_version=5.8.4
+pkg_version=5.10.3
 pkg_description="Artifactory is an advanced binary repository manager for use by build tools (like Maven and Gradle), dependency management tools (like Ivy and NuGet) and build servers (like Jenkins, Hudson, TeamCity and Bamboo).
 Repository managers serve two purposes: they act as highly configurable proxies between your organization and external repositories and they also provide build servers with a deployment destination for your internally generated artifacts."
 pkg_upstream_url=https://www.jfrog.com/artifactory/
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=("JFrog Artifactory EULA")
 pkg_source=https://dl.bintray.com/jfrog/${pkg_name}/org/artifactory/pro/jfrog-${pkg_name}/${pkg_version}/jfrog-${pkg_name}-${pkg_version}.zip
-pkg_shasum=c87be150f32ac32d35c87a533f65e959b610630d019fc03f35884f823bb4ec1d
+pkg_shasum=7c7a39f8a0a0898300e0f548ee3b293ad0aa5e893871ecb453177dec2a09442f
 pkg_deps=(core/bash core/jre8)
 pkg_exports=(
   [port]=port
 )
 pkg_exposes=(port)
-pkg_svc_user="root"
```

Comment:
This might cause problems for people if they enabled auto-update for this package.
If files are written by artifactory, returning to `hab` user might lead to permission issues.
Do you know if this is an actual problem?

Reply:
Hey @rsertelon , thanks for reviewing! So I feel at a bit of an impasse here. I would like to get this out of the root account but you're right, we may see permissions issues if anybody did an auto-update. For now, I have reverted the account change but would like to reevaluate this down the road. Thanks!

Reply:
We've got the same problem with `core/jenkins` actually. We definitely need to do something about it. Maybe we should break packages once, the soonest, and then forbid a service from ever being run as root unless mandatory (privileged ports for example).